src/main/java/mate/academy/exception/NoUserInCacheException.java

```diff
@@ -0,0 +1,6 @@
+package mate.academy.exception;
+
+public class NoUserInCacheException extends RuntimeException {
+    public NoUserInCacheException() {
+    }
```

Comment:
The default constructor is empty. It is a good practice to provide a constructor that accepts a message or another Throwable as a parameter when creating custom exceptions. This can help to provide more context when the exception is thrown, making it easier to understand the issue.

Reply:
agree, exception should always have as detailed information as possible on what went wrong, so add exception and a message if you are going to use it